src/Text/Pandoc/Readers/RST.hs

```diff
@@ -770,6 +771,26 @@ tableDirective top _fields body = do
                                   aligns' widths' header' rows'
        _ -> return mempty
 
+listTableDirective :: PandocMonad m => String -> [(String, String)] -> String -> RSTParser m Blocks
+listTableDirective top _fields body = do
+  bs <- parseFromString parseBlocks body
+  title <- do { parseFromString (trimInlines . mconcat <$> many inline) top }
+  let rows = takeRows $ B.toList bs
+      numOfCols = length $ rows !! 0
```

Comment:
Note, `!!` is a partial function. This will fail with a runtime exception if `rows` is empty.  This case should be handled safely.